### DISCUSSION TITLE
Auto Dark Mode

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#82] [General]
# Auto Dark Mode

Check if Auto Dark Mode is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/ab58c7f7-af42-43a8-b27c-818c7ebafb81

**Comment by erikjoya** [comment]
There is a beta update that you can run after installing the app. This will install the native arm app

**Reply by OpenSource03** [reply]
Thanks. I updated the status.
